docs/Toy/toy/parser.py

```diff
@@ -447,5 +447,5 @@ def parseError(self,
         Location is retrieved from the lexer to enrich the error message.
         """
         token = self.getToken()
-        line = self.program.splitlines()[token.line]
+        line = self.program.splitlines()[token.line - 1]
```

Comment:
I think this needs a testcase, or maybe even its own PR.

Reply:
Added testcase, it's really a tiny change